src/collection/layout.js

```diff
@@ -74,6 +95,15 @@ var elesfn = ({
     } else {
       nodes.positions( fn );
 
+      if ( options.spacingFactor && options.spacingFactor !== 1){
+        var spacing = Math.abs(options.spacingFactor);
+        nodes.positions( function( i, node){
+          var pos = node.position();
```

Comment:
I think if the previous `.positions()` call is removed we can just have `pos` set by `fn()` with `fn()` called as described in http://js.cytoscape.org/#nodes.layoutPositions